hoomd/md/EvaluatorPairGB.h

```diff
@@ -192,6 +192,16 @@ class EvaluatorPairGB
             {
             return "gb";
             }
+
+        std::string getShapeSpec() const
+            {
+            std::ostringstream shapedef;
+            shapedef << "{\"type\": \"Ellipsoid\", \"a\": " << params.lpar <<
```

Comment:
Please make sure that the "parallel" axis is correctly chosen -- I'm not sure if it's supposed to be `a` or `c` (x-aligned or z-aligned in the particle frame). I read the docs and it wasn't clear to me which was the right choice.

Reply:
Yeah I wasn't 100% sure either. Have it in my notes to ask.

Reply:
@joaander can you clarify this?

Reply:
I think the parallel axis is z (c) with x and y (a and b) perpendicular. Can you confirm @jglaser?

Reply:
@joaander It seems that's the case, from the paper cited in the documentation ([Allen et al. 2006](https://www.tandfonline.com/doi/full/10.1080/00268970601075238))